datalake/src/main/java/com/sequenceiq/datalake/service/sdx/StructuredEventCleanUpService.java

```diff
@@ -0,0 +1,47 @@
+package com.sequenceiq.datalake.service.sdx;
+
+import java.util.Optional;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.cloudbreak.structuredevent.service.db.CDPStructuredEventDBService;
+import com.sequenceiq.cloudbreak.util.TimeUtil;
+
+@Service
+public class StructuredEventCleanUpService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(StructuredEventCleanUpService.class);
+
+    private static final int THREE_MONTHS = 3;
+
+    private final SdxService service;
+
+    private final TimeUtil timeUtil;
+
+    private final CDPStructuredEventDBService structuredEventDBService;
+
+    public StructuredEventCleanUpService(SdxService service, TimeUtil timeUtil, CDPStructuredEventDBService structuredEventDBService) {
```

Comment:
I know it's a matter of taste, but we usually don't use constructor-based injection in DL service. As I see you are not using the constructor in tests, so you can easily remove it and add @Inject to the 3 services which are injected.

Reply:
yeah it's my taste but I obey the general way of use in the service so replaced it with annotation injection :)

Reply:
thank you! :)